client/app/components/routes/operations/Page.tsx

```diff
@@ -2,12 +2,12 @@
 import Link from "next/link";
 import { Button } from "@mui/material";
 import { Suspense } from "react";
-import Operations from "@/app/components/routes/operations/Operations";
 import Loading from "@/app/components/loading/SkeletonGrid";
 import { getServerSession } from "next-auth";
 import { authOptions } from "@/app/api/auth/[...nextauth]/route";
 import Note, { registrationRequestNote } from "../../datagrid/Note";
-import { OperationsSearchParams } from "@/app/components/routes/operations/types";
+import { OperationsSearchParams } from "../../operations/types";
+import Operations from "../../operations/Operations";
```

Comment:
I think we should avoid using relative imports. Using it when it's in the same directory ie `./Operations` isn't too bad though I think going up multiple directories is confusing.  

Suggest to change all relative imports to aliased imports which are easier to read and more informative ie `@/app/components/operations/Operations`.

Reply:
I've fixed all the multiple directories ones but left ./

Reply:
I can still see about 6 imports in the changee files here that use `../../componentname`, other than that this is good to go!